packages/manager/modules/pci/src/projects/project/instances/instances.routing.js

```diff
@@ -6,8 +6,10 @@ import map from 'lodash/map';
 import { TYPES_TO_EXCLUDE } from './instances.constants';
 import Instance from '../../../components/project/instance/instance.class';
 
+const stateName = 'pci.projects.project.instances';
+
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('pci.projects.project.instances', {
+  $stateProvider.state(stateName, {
```

Comment:
Any reason for this change ?

Reply:
No need it now.
I changed the implementation.